### DISCUSSION TITLE
Link A Repo To A Project

### DISCUSSION THREAD

**Discussion by KarlDaw** [community/community#126553] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

I'm not sure if I just can't find this, but there doesn't seem to be a way to link a repo to a project. The Issues view has been massively useful, as we can use Issues to plan out sprints using the Project feature, however when going from the project to a repo we have to go out of the project view, then into the repos list to find the one we need.

Is there an easier way I have managed to overlook? 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by byambajav-m** [comment]
Here's how to do it in just a few clicks:

-Navigate to your repository on GitHub.
-Click on "Projects".
-Select "Link a project" and search for the relevant project.
-Click on the project to link it to your repository.

**Reply by KarlDaw** [reply]
Thanks for the quick response. I already have the repo linked to the project, the problem is once I'm on the projects board view, there is no way of going directly to the repo. The only way without exiting the view that I can find is to create a branch from an issue then use that branch to navigate, but if there are no branches on the issues I'm viewing there's no quick link to the repo.

I was hoping there would be a way to have this visible but by the looks of it it's something that's missing from the Projects view

**Comment by dhruv8433** [comment, marked as answer]
@KarlDaw 

As of now, GitHub Projects do not have a direct feature to link a repository in a way that allows one-click navigation from the project to the repository. However, you can add a reference to the repository in the project description or project notes for easier access.

When creating or editing issues directly from the project board, you can select the repository where the issue belongs. This can help you indirectly navigate to the repository since the issue will be linked to it.

Steps to Add Repository Reference in Project Description:

1. Navigate to your GitHub Project.
2. Click on the "..." (three dots) button in the top right corner of the project page.
3. Select "Edit".
4. In the description or body of the project, add a link to your repository. For example, you can add Repository Name(https://github.com/user/repo).

Example:
If your repository is located at https://github.com/user/repo, you can add the following markdown in the project description:

`For more details, visit the [Repository Name](https://github.com/user/repo).`

This way, anyone viewing the project can easily click the link to navigate to the repository.

**Reply by KarlDaw** [reply]
Thanks, it's a bit disappointing that there's no direct link but this workaround will get me what I want in the projects, thanks for the detailed steps.

**Comment by Fireye04** [comment]
I would definitely appreciate this being an actual integrated feature later down the line in a more mature version of projects. Having full repo integration from the development side of things would make project management especially on smaller teams so much easier. Additionally from user end, having an easily accessible roadmap would also be a really cool side effect of this. Hope that this is planned at some point <3

**Comment by Hritickjha** [comment]
Response #2

**Reply by Hritickjha** [reply]
Answer

**Comment by govarney** [comment]
This doesn't work on GitHub Enterprise, or at least the version we are running (3.12)
